library/src/commonMain/kotlin/org/jraf/klibnotion/internal/api/model/search/ApiSearchParametersConverter.kt

```diff
@@ -29,29 +29,22 @@ import org.jraf.klibnotion.internal.api.model.ApiConverter
 import org.jraf.klibnotion.model.property.sort.PropertySort
 
 internal object ApiSearchParametersConverter :
-    ApiConverter<
-            ApiSearchParameters,
-            Triple<String?, PropertySort?, String>
-            >() {
-    override fun modelToApi(model: Triple<String?, PropertySort?, String>): ApiSearchParameters {
-        val query = model.first
-        val sort = model.second
-        val type = model.third
-
+    ApiConverter<ApiSearchParameters, SearchParameters>() {
+    override fun modelToApi(model: SearchParameters): ApiSearchParameters {
         return ApiSearchParameters(
-            query = query,
-            sort = sort?.let {
+            query = model.query,
```

Comment:
We should also have `startCursor = model.startCursor,` here, right?

Reply:
Good catch!